src/main/java/WayofTime/bloodmagic/api/event/BoundToolChargeEvent.java

```diff
@@ -0,0 +1,18 @@
+package WayofTime.bloodmagic.api.event;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+@Cancelable
+public class BoundToolChargeEvent extends Event {
```

Comment:
Move all these events into a `BoundToolEvent` class. So `BoundToolEvent.Charge` and `BoundToolEvent.Release`